src/openMVG/sfm/pipelines/localization/SfM_Localizer.cpp

```diff
@@ -161,7 +155,8 @@ bool SfM_Localizer::RefinePose
   // pose
   sfm_data.poses[0] = pose;
   // intrinsic (the shared_ptr does not take the ownership, will not release the input pointer)
-  sfm_data.intrinsics[0] = std::shared_ptr<cameras::IntrinsicBase>(intrinsics, [](cameras::IntrinsicBase*){});
+  std::shared_ptr<cameras::IntrinsicBase> localIntrinsics(intrinsics->clone());
```

Comment:
FYI, camera intrinsic can be shared (with existing registered pose), so the clone is performs to avoid to modify a shared intrinsic.
If there is not clone, you will modify the intrinsic based only on one camera pose for all the view that shared this intrinsic parameters.